src/host/main.cpp

```diff
@@ -142,6 +142,8 @@ int main(int argc, char** argv)
     json_log_path,
     "Path to file where the json logs will be written");
 
+  bool json_log_std_out = false;
+  app.add_flag("--json-std-out", json_log_std_out, "send json to std out");
```

Comment:
Nitpick: Better to move the option next to `--json-log-path`.  And also perhaps rename it to `--log-format-json` and description to "Set node stdout log format to JSON".